force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,23 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String helloStr = 'Hello, Salesforce!';
+        // Get the length of the String      
+        Integer intStrLength = helloStr.length();
+        String strLength = String.valueOf(intStrLength);
+
         // Get a substring of the String of the first 5 characters
+        String firstFiveChars = helloStr.substring(0, 5);
         // Convert the String to lower case
+        String lowerCaseStr = helloStr.toLowerCase();            
         
         // Return the list with the string data
         // In the first position add the string length
         // In the second position add the substring
         // In the third position add the string in lower case
+        List<String> returnStr = new List<String>{strLength, firstFiveChars, lowerCaseStr};
 
-        return null; // Replace null with the variable you used to store the result
+        return returnStr;
```

Comment:
Good work. Another option is to assign these values to the list directly. I would do that if I am not using those variable for any other purpose. 

```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```